docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1050 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
+should have a `results` folder that looks like this:
+
+```console
+results
+├── multiqc
+│   ├── multiqc_data
+│   └── multiqc_report.html
+├── pipeline_info
+│   ├── execution_report_2024-11-14_12-07-43.html
+│   ├── execution_report_2024-11-14_12-12-42.html
+│   ├── execution_report_2024-11-14_12-13-58.html
+│   ├── execution_report_2024-11-14_12-28-59.html
+│   ├── execution_timeline_2024-11-14_12-07-43.html
+│   ├── execution_timeline_2024-11-14_12-12-42.html
+│   ├── execution_timeline_2024-11-14_12-13-58.html
+│   ├── execution_timeline_2024-11-14_12-28-59.html
+│   ├── execution_trace_2024-11-14_12-07-43.txt
+│   ├── execution_trace_2024-11-14_12-12-42.txt
+│   ├── execution_trace_2024-11-14_12-13-58.txt
+│   ├── execution_trace_2024-11-14_12-28-59.txt
+│   ├── params_2024-11-14_12-07-44.json
+│   ├── params_2024-11-14_12-12-43.json
+│   ├── params_2024-11-14_12-13-59.json
+│   ├── params_2024-11-14_12-29-00.json
+│   ├── pipeline_dag_2024-11-14_12-07-43.html
+│   ├── pipeline_dag_2024-11-14_12-12-42.html
+│   ├── pipeline_dag_2024-11-14_12-13-58.html
+│   ├── pipeline_dag_2024-11-14_12-28-59.html
+│   └── pipeline_software_mqc_versions.yml
+└── seqtk
+    ├── SAMPLE1_PE_sample1_R1.fastq.gz
+    ├── SAMPLE1_PE_sample1_R2.fastq.gz
+    ├── SAMPLE2_PE_sample2_R1.fastq.gz
+    ├── SAMPLE2_PE_sample2_R2.fastq.gz
+    ├── SAMPLE3_SE_sample1_R1.fastq.gz
+    └── SAMPLE3_SE_sample2_R1.fastq.gz
+```
+
+The resulting files of `multiqc` and `seqtk` are published respective subdirectories. In addition, `nf-core` pipelines by default have all sorts of reporting switched on. These files are stored in the `pipeline_info` subdirectory and time-stamped so that multiple runs don't overwrite them.
+
+### Handle modules output
+
+As with the inputs, you can view the outputs for the module by opening the `/modules/nf-core/seqtk/trim/main.nf` file and viewing the module metadata.
+
+```groovy title="modules/nf-core/seqtk/trim/main.nf" linenums="13"
+output:
+tuple val(meta), path("*.fastq.gz"), emit: reads
+path "versions.yml"                , emit: versions
+```
+
+To help with organization and readability it is beneficial to create named output channels.
+
+For `SEQTK_TRIM`, the `reads` output could be put into a channel named `ch_trimmed`.
+
+```groovy title="workflows/myfirstpipeline.nf" linenums="35"
+ch_trimmed  = SEQTK_TRIM.out.reads
+```
+
+Similarly, it is beneficial immediately mix the versions of tools into the `ch_versions` channel so they can be used as an input for the `MULTIQC` process.
+
+```groovy title="workflows/myfirstpipeline.nf" linenums="35"
+ch_versions = ch_versions.mix(SEQTK_TRIM.out.versions.first())
+```
+
+!!! note
+
+    The `first` operator is used to emit the first item from `SEQTK_TRIM.out.versions` to avoid duplication.
+
+### Add a parameter to the `seqtk/trim` tool
+
+To prevent changing the nf-core modules, additional configuration options can be applied to a module using scopes within configuration files.
```

Comment:
```suggestion
To prevent changing the nf-core modules, additional configuration options can be applied to a module using scopes within configuration files.
```

This probably needs more explanation. Not sure how though